packages/flutter/test/cupertino/page_test.dart

```diff
@@ -222,7 +228,7 @@ void main() {
     tester.state<NavigatorState>(find.byType(Navigator)).pushNamed('/next');
 
     await tester.pump();
-    await tester.pump(const Duration(milliseconds: 400));
+    await tester.pumpAndSettle();
```

Comment:
I'd still keep a concrete number here. That the animation doesn't go on for 2s is still part of the 'test'.